ad/persistence/ad-cs/certificate-authority.md

```diff
@@ -29,7 +29,7 @@ Then, forging (and signing) a certificate can be done as follows.
 
 {% code overflow="wrap" %}
 ```bash
-certipy forge -ca-pfx "CA.pfx" -upn "administrator@corp.local" -subject "CN=Administrator,CN=Users,DC=CORP,DC=LOCAL"
+certipy forge -ca-pfx "CA.pfx" -upn "administrator@corp.local" -subject "CN=Administrator,CN=Users,DC=CORP,DC=LOCAL" (-crl "ldap://...")
```

Comment:
in my opinion it would be better to have one info/tip bubble right before the commands. "By using the `--crl 'ldap://...'`, certipy can blablabla".

Reply:
ouaip c'est better